src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -971,13 +975,26 @@ export default class ComparisonStore {
     });
 
     readonly survivalClinicalDataExists = remoteData<boolean>({
-        await: () => [this.activeSamplesNotOverlapRemoved],
+        await: () => [
+            this.activeSamplesNotOverlapRemoved,
+            this.survivalClinicalAttributesPrefix,
+        ],
         invoke: async () => {
             if (this.activeSamplesNotOverlapRemoved.result!.length === 0) {
                 return false;
             }
+            const attributeNames: string[] = [];
+            _.reduce(
```

Comment:
reduce should always return the product of the operation.   it should be attributeNames = _.reduce(...)